packages/kbn-telemetry-tools/src/cli/run_telemetry_check.ts

```diff
@@ -59,11 +67,28 @@ export function runTelemetryCheck() {
           title: 'Extracting Collectors',
           task: (context) => new Listr(extractCollectorsTask(context, path), { exitOnError: true }),
         },
+        {
+          enabled: () => typeof path !== 'undefined',
+          title: 'Checking collectors in --path are not excluded',
+          task: ({ roots }: TaskContext) => {
+            const totalCollections = roots.reduce((acc, root) => {
+              return acc + (root.parsedCollections?.length || 0);
+            }, 0);
+            const collectorsInPath = Array.isArray(path) ? path.length : 1;
+
+            if (totalCollections !== collectorsInPath) {
+              throw new Error(
+                'Collector specified in `path` is excluded; Check the telemetryrc.json files.'
+              );
+            }
+          },
+        },
```

Comment:
Just to fully understand this piece of code: Each `path` should only contain one collector for this to work. i.e.: I can't run `--path=src/plugins/kibana_usage_collection`

Is that correct?

Reply:
yes. It is documented in the README as well:

> * `--path` specifies a collector path instead of checking all collectors specified in the `.telemetryrc.json` files. Accepts a `.ts` file. The file must be discoverable by at least one rc file.